sdk/keyvault/tests.yml

```diff
@@ -6,6 +6,7 @@ extends:
     ServiceDirectory: keyvault
     MaxParallel: 5
     TimeoutInMinutes: 180
+    Clouds: 'Public,UsGov,China,Canary'
```

Comment:
This should be [SupportedClouds](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/389/Configuring-Tests?anchor=cloudconfig-schema) as we don't want to run sovereign clouds in the nightly builds.

Reply:
I have changed Clouds to SupportedClouds.